logger/logger.py

```diff
@@ -0,0 +1,42 @@
+import logging
+import functools
+import os
+from flask import request
+
+def get_logger(name, log_file, level=logging.INFO):
+    log_directory = os.path.join(os.path.dirname(__file__), "logs")
+
+    os.makedirs(log_directory, exist_ok=True)
+
+    log_file_path = os.path.join(log_directory, log_file)
+
+    logger = logging.getLogger(name)
+
+    if not logger.hasHandlers():
+        handler = logging.FileHandler(log_file_path, mode='w')
+        formatter = logging.Formatter("%(asctime)s - %(levelname)s - %(funcName)s - %(message)s")
+        handler.setFormatter(formatter)
+        logger.addHandler(handler)
+        logger.setLevel(level)
+
+    return logger
+
+
+def log_function_call(logger):
+    def decorator(func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            try:
+                json_data = request.get_json(silent=True)
+                logger.info(f"Call of {func.__name__}() з args={args}, kwargs={kwargs}, json_data={json_data}")
+
+                result = func(*args, **kwargs)
+
+                logger.info(f"Result of {func.__name__}() = {result}")
```

Comment:
100% Debug. Or even better to have this removed.

We only log something that may be useful later and while I agree that function invokation and its results are indeed useful - if we log them all then all our log becomes bloody unreadable.

So it's a balance between having as much as possible without having it all together